### ISSUE TITLE
Name property for List View element is not define

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#332]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to List view item in 'Create new Workbook' and observe Inspect property

### Expected Behavior
Name property should be clearly defined for List View  item
### Actual Behavior
Name property for list view item is not defined and currently is Empty

**Narrator behavior:** Narrator is not announcing the name of list view and it is announcing the list item selected in List view as :' Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem selected, selection contains <spell>4</spell> items,
0% vertically scrolled'
**NVDA behavior:** NVDA is announcing as 'Xamarin workbook window, list, list'

### User Impact
Users who rely on screen reader will not be able to know the Name of List View item 

![list_view_name_property_inspect](https://user-images.githubusercontent.com/22956130/40720287-dd1d11e0-6433-11e8-9611-1e8fc59beadd.png)
![list_view_name_property_narrator](https://user-images.githubusercontent.com/22956130/40720330-013c7db8-6434-11e8-8759-ced6cac73563.png)
![list_view_name_property_nvda](https://user-images.githubusercontent.com/22956130/40720349-0d075712-6434-11e8-923b-d7c0b2c3d6c1.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbook-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
